app/data_collection/blog_data.py

```diff
@@ -0,0 +1,166 @@
+            for href in site_href_list
+            if "/news" in href and href != "/news"
+        ]
+        return list(set(news_hrefs))
+
+    def get_ncsoft_suburls(self) -> list:
+        """
+        NCSoft 블로그 페이지에서 모든 게시물의 URL을 가져옵니다.
+
+        Returns:
+            list: 모든 블로그 게시물의 URL을 포함한 리스트를 반환합니다.
+        """
+        url = "https://ncsoft.github.io/ncresearch/blogs/"
+        site_href_list = self.get_all_hrefs(url)
+        pattern = re.compile(r"^/ncresearch/[a-fA-F0-9]{40}$")
+        filtered_paths = [
+            url.replace("/ncresearch/blogs/", "") + path
+            for path in site_href_list
+            if pattern.match(path)
+        ]
+        return filtered_paths
+
+    def get_naver_suburls(self) -> list:
+        """
+        Naver Tech Blog에서 모든 게시물의 URL을 가져옵니다.
+
+        Returns:
+            list: 모든 게시물의 URL을 포함한 리스트를 반환합니다.
+        """
+        url = "https://clova.ai/tech-blog"
+        site_href_list = list(set(self.get_all_hrefs(url)))
+        filtered_path = [
+            url.replace("/tech-blog", "") + path
+            for path in site_href_list
+            if ("/tech-blog" in path)
+            and ("/tag/" not in path)
+            and (path != "/tech-blog")
+        ]
+        return filtered_path
+
+    def get_all_docs(self) -> list:
+        """
+        지정된 회사들 (OpenAI, Anthropic, NCSoft, Naver) 웹사이트에서 모든 문서를 가져옵니다.
+
+        Returns:
+            list: 모든 문서를 담고 있는 리스트를 반환합니다.
+        """
+        # 모든 문서 서브 URL을 통해 문서 목록을 가져옵니다
+        # openai = self.get_openai_suburls()
+        anthropic = self.get_anthropic_suburls()
+        ncsoft = self.get_ncsoft_suburls()
+        naver = self.get_naver_suburls()
+
+        total_list_suburl = anthropic + ncsoft + naver
+
+        loader = WebBaseLoader(total_list_suburl)
+        docs = loader.load()
+        return docs
+
+
+'''
+    def get_openai_suburls(self, n_recent: int = 30) -> list:
+        """
+        OpenAI 웹사이트에서 최근 게시된 뉴스 페이지들의 URL을 반환합니다.
+
+        Args:
+            n_recent (int): 최근에 게시된 게시물 중에서 가져올 개수입니다.
+
+        Returns:
+            list: 최근에 추가된 뉴스 페이지들의 URL을 포함한 리스트입니다.
+        """
+        url = f"https://openai.com/news/?limit={n_recent}"
+        openai_url_list = list()
+        try:
+            self.driver.get(url)
+
+
+            # Javascript가 로드될 수 있도록 기다리는 시간을 설정합니다
+            time.sleep(2)
+
+            # Find all <a> tags
+            links = self.driver.find_elements(By.TAG_NAME, "a")
+            print(links)
+            for link in links:
+                link_dir = link.get_attribute("href")
+                if "/index/" in link_dir:
+                    openai_url_list.append(link_dir)
+        finally:
+            # 크롤링 완료 후 driver 종료합니다
+            self.driver.quit()
+
+        return list(set(openai_url_list))
+'''
+
+if __name__ == "__main__":
+    website_data = WebsiteDataCrawler()
+    site_list = website_data.get_all_docs()
+
+    for site in site_list:
+        print(site)
```

Comment:
이렇게 모듈을 직접 실행 할 수 있게 구현하셨다면 실행 커맨드와 실행시 어떤 작업을 하는지 주석을 추가해주면 좋습니다.
아래는 예시입니다!

```suggestion
if __name__ == "__main__":
    """ 
    A, B, C를 해서 사이트 목록을 출력
    
    usage: python -m app.data_collection.blog_data
    """        
    website_data = WebsiteDataCrawler()
    site_list = website_data.get_all_docs()

    for site in site_list:
        print(site)
```

Reply:
[47a2c1f](https://github.com/f-lab-edu/llm-mlops/pull/2/commits/47a2c1fbb6466583d8df056c20f5b8148547c780)에 수정했습니다!